packages/git-encrypted/src/keyDerivation.ts

```diff
@@ -0,0 +1,15 @@
+import { scrypt } from 'scrypt-js';
+
+export const hash = async (password: string, salt: string) => {
```

Comment:
Great that this works! Thanks for pushing this forward.

Could we incorporate the `N`, `r`, `p`, and `dkLen` as input to the function?

My thinking is that these params will all be written into a file, unencrypted, along with the `salt`, into the root of the encrypted repository. That way a user could change these params, and still be able to generate the same keys.

Reply:
Sure. `dkLen` is the output length, that should always stay at 96 bytes no?

Reply:
@johannes87 You're right, `dkLen` can be a constant. I suppose that if we change the number of keys (and so the output length) we'll need to figure that out in a different manner. 👍